### ISSUE TITLE
Failed to build UWP Project for two files missed

### ISSUE THREAD

**Issue by evolcano** [Baseflow/XamarinMediaManager#722]
## 🐛 Bug Report

After change the targeting of UWP project to latest version, project build failed for miss VideoView.xaml and MediaManager.xr.xml.
VideoView.xaml actually existed in "C:\Users\HMY\.nuget\packages\plugin.mediamanager\0.9.8-alpha2\lib\uap10.0.17763\MediaManager\Platforms\Uap\Video", not "C:\Users\HMY\.nuget\packages\plugin.mediamanager.forms\0.9.8-alpha2\lib\uap10.0.17763\MediaManager\"

### Expected behavior
build or debug success

### Reproduction steps
1 Create .net standard 2.0 class library project , and add reference to Plugin.MediaManager.Forms
2 Add code using VideoView for video play
3 Add Xamarin projects to SLN include UWP project
4 add class library reference to UWP project
5 change UWP project Target version and MIn version both to windows version 1903(build 18362)
5 Select Debug or RELEASE, run or build

### Configuration
OS: windows 10
Version of VS: 16.6.1
Version of Xamarin: 4.6.0.847+487-sha.6f0fabeba-azdo.3764095
UWP target version: 1903(build 18362)

**Version:** 0.9.8-alpha2

**Platform:** 
- [ ] :iphone: iOS
- [ ] :robot: Android
- [ ] :checkered_flag: WPF
- [* ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

###Build output
```
3>F:\vs2019\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "C:\Users\HMY\.nuget\packages\plugin.mediamanager.forms\0.9.8-alpha2\lib\uap10.0.17763\MediaManager\Platforms\Uap\Video\VideoView.xaml" because it was not found.
3>F:\vs2019\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "C:\Users\HMY\.nuget\packages\plugin.mediamanager.forms\0.9.8-alpha2\lib\uap10.0.17763\MediaManager\MediaManager.xr.xml" because it was not found.
========== Build: 2 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

**Comment by Agredo** [comment]
I could solve this by setting Plugin.MediaManager to 0.9.8 alpha 3 and Plugin.MediaManager.Forms to 0.9.7 (0.9.8. alpha 2 did caused this files missing error). But Currently the player does not appears but no errors occurring!

**Comment by evolcano** [comment]
Yes, I copied files. Thanks!

**Comment by xabre** [comment]
UWP Files are still missing in the latest 1.0.3 release. 
Was able to workaround the issue in a similar way as above by combining versions: **Plugin.MediaManager v1.0.3** and **Plugin.MediaManager.Forms v0.9.7**.

**Comment by smlsr** [comment]
UWP files are still missing in 1.0.8?  Can not build in AppCenter

**Comment by martijn00** [comment]
It does exist. You can see it when you open the package. Maybe You need to update your visual studio?

<img width="712" alt="Screenshot 2021-01-08 at 17 16 13" src="https://user-images.githubusercontent.com/6221156/104038332-45734c00-51d5-11eb-8224-343a2814b411.png">

This is the code that adds it: https://github.com/Baseflow/XamarinMediaManager/blob/develop/MediaManager/MediaManager.csproj#L16

**Comment by smlsr** [comment]
The error is in MediaManager.Forms it does not exist and fails in AppCenter (and locally as well)

If I copy the XR.xml from MediaManager, create the MediaManager folder in plugging.mediamanager.forms, and then place file in that folder will build locally

Shawn

**Comment by SiliconBoy007** [comment]
I experienced the exact same issue in the MediaManager.Forms for UWP and had to copy the XR file as suggested to fix the problem on version 1.0.8
